go/vt/vttablet/endtoend/queries_test.go

```diff
@@ -1789,6 +1789,38 @@ func TestQueries(t *testing.T) {
 				},
 			},
 		},
+		&framework.MultiCase{
+			Name: "impossible queries",
+			Cases: []framework.Testable{
+				&framework.TestCase{
+					Name:  "specific column",
+					Query: "select eid from vitess_a where 1 != 1",
+					Rewritten: []string{
+						"select eid from vitess_a where 1 != 1",
+					},
+					RowsAffected: 0,
+				},
+				&framework.TestCase{
+					Name:  "all columns",
+					Query: "select * from vitess_a where 1 != 1",
+					Rewritten: []string{
+						"select * from vitess_a where 1 != 1",
```

Comment:
Backing out the fix in this PR makes another query show up here, which is nice to see.